### PR TITLE
Changes for CSR and AKS

### PR DESCRIPTION
**Pull request by jojimt**

### CODE REVIEW

File: provision/acc_provision/templates/aks-provision-config.yaml [hunk 1]
```diff
@@ -9,22 +9,30 @@ aci_config:
...
+  node_subnet: 10.0.0.1/24      # must be part of machine_cidr
+  extern_dynamic: 10.3.0.1/24   # Subnet to use for dynamic external IPs
+  extern_static: 10.4.0.1/24    # Subnet to use for static external IPs
+  node_svc_subnet: 10.5.0.1/24  
   opflex_server_port: 19999 
   interface_mtu: 1450          # min = 1280 for ipv6, max = 8900 for VXLAN
+  #service_monitor_interval: 5  # IPSLA interval probe time for PBR tracking
+                                # default is 5, set to 0 to disable, max: 65535
+  #pbr_tracking_non_snat: true  # Default is false, set to true for IPSLA to
+                                # be effective with non-snat services
```
Comment (snaiksat) [inline provision/acc_provision/templates/aks-provision-config.yaml:35]: Are these two relevant to AKS?

Comment (jojimt) [reply provision/acc_provision/templates/aks-provision-config.yaml:35]: They are not relevant. Even if user adds, it will be ignored. It's there only because of the common template.

**Review by snaiksat — APPROVED** [review]
